kDriveCore/Utils/PHAsset/PHAssetIdentifier.swift

```diff
@@ -147,32 +147,43 @@ struct PHAssetIdentifier: PHAssetIdentifiable {
             let activity = ExpiringActivity(id: uid, delegate: activityDelegate)
             activity.start()
 
-            // TODO: Check iCloud behaviour
             let options = PHAssetResourceRequestOptions()
             options.isNetworkAccessAllowed = true
             options.progressHandler = { progress in
                 Log.photoLibraryUploader("hashing resource \(progress * 100)% …")
             }
 
             group.enter()
+            var resourceManagerError: Error?
             PHAssetResourceManager.default().requestData(for: bestResource,
                                                          options: options) { data in
                 hasher.update(data)
-            } completionHandler: { _ in
+            } completionHandler: { error in
```

Comment:
I now capture the error if we fail to get an asset to hash it during the library scanning.